public/doc/openapi-3-v1.2.json

```diff
@@ -1518,6 +1504,75 @@
                         "description": "This is the offset from beginning of the result set"
                     }
                 }
+            },
+            "400BadRequestResponse": {
+                "type": "object",
+                "description": "Bad request: either all the required parameters specified or the parameter value is invalid.",
+                "properties": {
+                    "errors": {
+                        "type": "Array",
+                        "description": "Errors resulting from the API query",
+                        "items": {
+                            "type": "object",
+                            "properties": {
+                                "status": {
+                                    "type": "integer",
+                                    "example": 400
```

Comment:
@hsong-rh We are seeing conflicts here some of the status fields have the value bad_request instead of the integer 400.